index.html

```diff
@@ -204,40 +171,37 @@ <h2>Photo Gallery</h2>
 <section class="survey">
     <h2>Survey</h2>
     <form method="POST" action="https://formspree.io/mgevdyal">
-        <label>What is your name?<br>
-         <input name="name_feild" type="text" required>
```

Comment:
You were asked to use the `for` attribute for labels (except for the radio button) but you did it backwards